ebuild-writing/bundled-dependencies/text.xml

```diff
@@ -0,0 +1,373 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<devbook self="ebuild-writing/bundled-dependencies/">
+<chapter>
+<title>Bundled dependencies</title>
+
+<body>
+<p>
```

Comment:
"Blank lines must be placed immediately after every `<body>` (opening tag only) and before every `<section>`, `<p>`, `<pre>`, `<codesample>`, `<figure>`, `<table>`, `<ul>`, `<ol>`, `<dl>`, `<note>`, `<important>` and `<warning>` (opening tags only)."
```suggestion
<title>Bundled dependencies</title>
<body>

<p>
```